apim/3.x/templates/api/api-configmap.yaml

```diff
@@ -345,9 +345,9 @@ data:
           {{- end }}
         {{- end }}
     # SMTP configuration used to send mails
+    {{- if .Values.smtp.enabled }}
```

Comment:
Wondering it we should not rely on `.Values.smtp` only, in case the APIM admin wants to disable the smtp and blocks its activation in the UI (would be `email.enabled: false`)

Reply:
Also, can you add the Jira + GH Issues links in the commit message 😇